client/src/components/new-request/search.vue

```diff
@@ -1,89 +1,163 @@
-                        class="copy-small px-0 mx-0"
-                        label="The name is a person's name" />
-        </template>
-        <p class="py-0 my-0">Check this box if you are...</p>
+        <v-tooltip bottom content-class="bottom-tooltip mt-n5">
+          <template v-slot:activator="{ on }">
+            <v-checkbox
+                    v-model="isPersonsName"
+                    id="name-checkbox"
+                    class="copy-small mr-5"
+                    v-slot:label v-on="on">
+              <template>
+                <span v-on="on">Name is a person's name</span>
+              </template>
+            </v-checkbox>
+          </template>
+          <p>Check this box if you are...</p>
           <ul>
             <li>Incorporating under your own name (eg. DR. JOE SMITH INC.)</li>
             <li>The name contains one or more names. (eg. BLAKE, CHAN & DOUGLAS INC.)</li>
-            <li>The name contains one or more names. (eg. FRANKLIN INC.)</li>
           </ul>
-      </v-tooltip>
-      </v-col>
-      <v-col cols="3">
-      <v-tooltip bottom open-delay="300">
-        <template v-slot:activator="{on}">
-
-            <v-checkbox v-model="nameIsEnglish"
-                        v-on="on"
-                        id="name-checkbox"
-                        class="copy-small ml-n6"
-                        label="The name is English" />
-
-        </template>
-        <p class="py-0 my-0">This refers to the language of the words in your name.</p>
-        <ul>
-          <li>Leave this box checked if your name contains <b>only</b> English <b>or a mix</b> of English and
-            another Language
-          </li>
-          <li>Uncheck this box if your name is written <b>entirely</b> in another language and does <b>not</b> contain
-            any English
-          </li>
-        </ul>
-      </v-tooltip>
+        </v-tooltip>
       </v-col>
-      <v-col cols="5">
-        <span id="nr-required-activator"
-              class="link-std"
-              style="margin-left: auto"
-              @click="activateNRRModal()">Check to see if you need to file a a name request</span>
+      <v-col cols="4">
+        <v-tooltip bottom content-class="bottom-tooltip mt-n5">
+          <template v-slot:activator="{ on }">
+            <v-checkbox
+                    v-model="nameIsEnglish"
+                    id="name-checkbox"
+                    :false-value="true"
+                    :true-value="false"
+                    class="copy-small mr-5"
+                    v-slot:label v-on="on">
+              <template>
+                <span v-on="on">Name contains no English words</span>
+              </template>
+            </v-checkbox>
+          </template>
+          <p>This refers to the language of the words in your name.</p>
+          <ul>
+            <li>Leave this box checked if your name contains <b>only</b> English <b>or a mix</b> of English and
+              another Language
```

Comment:
"Language" should not be capitalized (IMHO).

Leave it if design has it this way.

Reply:
ah good catch, will fix!

Reply:
fixed!